### ISSUE TITLE
OCI Runtime create failed

### ISSUE THREAD

**Issue by konne** [gitpod-io/gitpod#8070]
### Bug description

I don't know if this is new or something related to #7666 

![image](https://user-images.githubusercontent.com/1100969/152788656-e95f2f19-768d-4eb8-a1ee-b26fc83ed281.png)

### Steps to reproduce

docker-compose up

### Workspace affected

https://vizlib-gucci-y03ugq7rbem.ws-eu30.gitpod.io/

**Comment by kylos101** [comment]
Hey @konne I am so sorry to sound like a broken record, but, is this something that happens consistently, or intermittently when you `docker-compose up`? I assume intermittent, but do not want to assume. Let us know?

**Comment by konne** [comment]
@kylos101 happens roughly about 10% of the starts. So we can as a workaround just restart docker-compose up and usually, it is working then but still, it is annoying.

**Comment by utam0k** [comment]
@konne Hi, we have made some improvements in this area(seccomp notify). 
https://github.com/gitpod-io/gitpod/pull/8004

Could you tell me if this still happens in an environment where this `ws-eu$(number)` part of the workspace's URL is set to 31 or 32? I tried to run the redis image multiple times in parallel using docker in 31 environments, and it worked without any problems. We are lucky if the above improvement cured this issue, but if it still happens, please let us know.

**Comment by kylos101** [comment]
Hey @konne :wave: , hope you're doing well! May I ask, is your team still experiencing this issue? I ask because we shipped a fix about 8 days ago which may resolve this issue.

**Comment by konne** [comment]
@kylos101 I'm this week Off. I will give you feedback next week after I talked with my devs.

**Comment by kylos101** [comment]
@konne , just wanted to circle back to this and see how things are going? Hope you had a nice vacation.

**Comment by konne** [comment]
@kylos101 I will close this ticket. I got the info from the team that they don't saw the issue in the last few days.
If it happens again, I will create a new ticket and link this one.
THX

**Comment by kylos101** [comment]
@konne fantastic news, thank you!
